TFproviderTest.yml

```diff
@@ -39,14 +40,19 @@ pipelines:
             - echo "Run Helm installation script" && ./get_helm.sh && rm get_helm.sh
             - helm version
             - apt update
-            - export GO_VERSION="1.17.8"
             - export GO_ARCH="amd64"
+            - rm -rf *.tar.gz go
```

Comment:
Yeah, I don't like this at all - I have no idea what dir this operates in and it removes all tgz - this is indescriminate and dangerous as a matter of principal. Please be much more explicit

Reply:
Use fully qualified paths here

Reply:
Mitigated